docs/src/components/ComponentDoc/ComponentDocAccessibility.tsx

```diff
@@ -21,17 +29,89 @@ function getStem(info) {
 }
 
 function getBehaviorName(stem) {
-  const filename = stem && `${stem}.ts`
+  if (!stem) {
+    return undefined
+  }
+  const filename = stem && `${stem}.ts`.replace(stem[0], stem.charAt(0).toLowerCase())
+  for (let i = 0; i < behaviorMenu.length; i++) {
+    const behaviour = behaviorMenu[i].variations.find(
+      behaviorVariation => behaviorVariation.name === filename,
+    )
+    if (behaviour) {
+      return behaviorMenu[i].displayName
+    }
+  }
+}
+
+function getId(idName: string) {
+  return _.kebabCase(idName)
+}
+
+function getAvailableBehaviorsFromJson(availableBehaviors: any): any[] {
```

Comment:
We have types for behaviors:

```suggestion
function getAvailableBehaviorsFromJson(availableBehaviors: BehaviorInfo): ???[] {
```

Reply:
thanks implemented and added new type for particular variant